docs/docusaurus.config.js

```diff
@@ -269,7 +265,7 @@ const config = {
           },
           {
             from: '/example-projects',
-            to: '/home/get-started#example-projects-repository'
+            to: '/#examples'
           },
```

Comment:
I'm not sure this is working correctly? 

<img width="1919" height="1016" alt="image" src="https://github.com/user-attachments/assets/c2d07d53-f0e6-48f5-b52e-2a4bd47a723c" />